scripts/setup-ubuntu.sh

```diff
@@ -31,43 +43,50 @@ DEPENDENCY_DIR=${DEPENDENCY_DIR:-$(pwd)}
 export CMAKE_BUILD_TYPE=Release
 SUDO="${SUDO:-"sudo --preserve-env"}"
 
-# Install all velox and folly dependencies.
-# The is an issue on 22.04 where a version conflict prevents glog install,
-# installing libunwind first fixes this.
-
-${SUDO} apt update
-${SUDO} apt install -y libunwind-dev
-${SUDO} apt install -y \
-  g++ \
-  cmake \
-  ccache \
-  ninja-build \
-  checkinstall \
-  git \
-  libc-ares-dev \
-  libcurl4-openssl-dev \
-  libssl-dev \
-  libicu-dev \
-  libdouble-conversion-dev \
-  libgoogle-glog-dev \
-  libbz2-dev \
-  libgflags-dev \
-  libgmock-dev \
-  libevent-dev \
-  liblz4-dev \
-  libzstd-dev \
-  libre2-dev \
-  libsnappy-dev \
-  libsodium-dev \
-  libthrift-dev \
-  liblzo2-dev \
-  libelf-dev \
-  libdwarf-dev \
-  bison \
-  flex \
-  libfl-dev \
-  tzdata \
-  wget
+# Install packages required for build.
+function install_build_prerequisites {
+  ${SUDO} apt update
+  # The is an issue on 22.04 where a version conflict prevents glog install,
+  # installing libunwind first fixes this.
+  ${SUDO} apt install -y libunwind-dev
+  ${SUDO} apt install -y \
+    g++ \
```

Comment:
```suggestion
    build-essential \
```
might be more accurate?

Reply:
Fixed